release/test_releases_properties.sh

```diff
@@ -0,0 +1,55 @@
+#!/bin/bash
+
+source _liferay_common.sh
+source _package.sh
+source _test_common.sh
+
+function main {
+	set_up
+
+	test_generate_release_properties_file_ga
+
+	LIFERAY_RELEASE_PRODUCT_NAME="dxp"
+	_PRODUCT_VERSION="7.4.13-u36"
+
+	test_generate_release_properties_file_u
+
+	tear_down
+}
+
+function set_up {
+	export LIFERAY_RELEASE_PRODUCT_NAME="portal"
+	export _BUNDLES_DIR="${PWD}"
+	export _PRODUCT_VERSION="7.4.3.120-ga120"
+
+	mkdir -p "${_BUNDLES_DIR}/tomcat"
+
+	echo "Apache Tomcat Version 9.9.99" > "${_BUNDLES_DIR}/tomcat/RELEASE-NOTES"
+}
+
+function tear_down {
+	unset LIFERAY_RELEASE_PRODUCT_NAME
+	unset _BUNDLES_DIR
+	unset _PRODUCT_VERSION
+
+	rm release.properties
+	rm -fr tomcat
+}
+
+function test_generate_release_properties_file_ga  {
+	generate_release_properties_file &>/dev/null
+
+	assert_equals \
+		"$(grep 'target.platform.version' release.properties | cut -d '=' -f 2)" \
+		"${_PRODUCT_VERSION}"
+}
+
+function test_generate_release_properties_file_u  {
```

Comment:
@kiwm, can you please rename `test_generate_release_properties_file_ga` and  `test_generate_release_properties_file_u` to `test_generate_release_properties_file_portal` and  `test_generate_release_properties_file_dxp`, respectively?

We're already using this terminology in https://github.com/liferay/liferay-docker/blob/f31542650d49126279c905502f534b6b6cb6f751/release/test_bom.sh#L15-L22

Thanks